newdle/client/src/components/Navigator.js

```diff
@@ -21,34 +23,56 @@ export default function Navigator() {
     },
   ];
   const active = useSelector(getStep);
+  const slots = useSelector(getFullTimeslots);
+  const dispatch = useDispatch();
   const activeStep = steps[active - 1];
 
   return (
     <Container>
-      {steps.map((step, index) => (
-        <Step key={index} active={index + 1 === active}>
-          {index + 1}
-        </Step>
-      ))}
+      <ul className={styles['step-box']}>
+        {steps.map((__, index) => (
+          <Step
+            key={index}
+            index={index + 1}
+            active={index + 1 === active}
+            // Step 3 (index === 2) should only be clickable if there are already slots defined
+            onClick={index !== 2 || !_.isEmpty(slots) ? () => dispatch(setStep(index + 1)) : null}
```

Comment:
Passing `null` doesn't use the default prop, only `undefined` does.

Reply:
Does it really matter, though? The child component doesn't misbehave when `onClick=null`.

Reply:
But OK, it's prettier, since otherwise the default value would never be used...

Reply:
no, i just noticed that you are specifying a default that's never used

Reply:
Yeah, I'm fixing it, thanks 😉